doc/modules/linear_model.rst

```diff
@@ -565,13 +565,13 @@ algorithm for approximating the fit of a linear model with constraints imposed
 on the number of non-zero coefficients (ie. the :math:`\ell_0` pseudo-norm).
 
 Being a forward feature selection method like :ref:`least_angle_regression`,
-orthogonal matching pursuit can approximate the optimum solution vector with a
+the orthogonal matching pursuit can approximate the optimum solution vector with a
 fixed number of non-zero elements:
 
 .. math::
     \underset{w}{\operatorname{arg\,min\,}}  ||y - Xw||_2^2 \text{ subject to } ||w||_0 \leq n_{\text{nonzero\_coefs}}
 
-Alternatively, orthogonal matching pursuit can target a specific error instead
+Alternatively,the orthogonal matching pursuit can target a specific error instead
```

Comment:
orthogonal matching pursuit  is the name of the method. The previous text was correct.